ndbench-core/src/main/java/com/netflix/ndbench/core/defaultimpl/NdBenchGuiceModule.java

```diff
@@ -47,10 +47,9 @@ protected void configure()
     {
         bind(IConfiguration.class).to(NdBenchConfiguration.class);
         bind(NdBenchMonitor.class).to(FakeMonitor.class);
-        bind(IClusterDiscovery.class).to(LocalClusterDiscovery.class);
+        bind(IClusterDiscovery.class).to(AWSLocalClusterDiscovery.class);
```

Comment:
Thanks for adding AWSLocalClusterDiscovery, instead of making it default binding, let's make it optional binding so that NDBench can startup on any cloud/ datacenter instances.